graphql_compiler/tests/schema_transformation_tests/input_schema_strings.py

```diff
@@ -101,7 +103,7 @@ class InputSchemaStrings(object):
           age: Int
         }
 
-        type Human implements Character, Creature {
+        type Human implements Character & Creature {
```

Comment:
This is super subtle but a massive compatibility improvement, it was such a struggle without this!